meta/main.yml

```diff
@@ -17,24 +17,12 @@ galaxy_info:
         - focal
 
   galaxy_tags:
-    - grafana-agent
+    - grafana
     - prometheus
     - loki
     - tempo
     - promtail
     - server
     - agent
-    - consul_exporter
-    - dnsmasq_exporter
-    - elasticsearch_exporter
-    - kafka_exporter
-    - memcached_exporter
-    - mysqld_exporter
-    - node_exporter
-    - postgres_exporter
-    - process_exporter
-    - redis_exporter
-    - statsd_exporter
-    - windows_exporter
```

Comment:
why did you delete these references? all this exporters are included.

Reply:
these are invalid tags according to ansible-lint (no underscore allowed)